roles/build_openstack_packages/tasks/append_repos.yml

```diff
@@ -5,6 +5,8 @@
   register: _repo_data
 
 - name: Append the content in mock config
-  ansible.builtin.lineinfile:
-    path: "{{ cifmw_bop_build_repo_dir }}/DLRN/scripts/{{ cifmw_bop_initial_dlrn_config }}-local.cfg"
-    line: "{{ _repo_data['content'] | b64decode }}"
+  ansible.builtin.set_fact:
```

Comment:
Maybe a nit but this task name should probably be updated since it's not appending anything anymore but storing a fact for later use